app/models/micropost.rb

```diff
@@ -0,0 +1,18 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
```

Comment:
association này có tác dụng gì em ?

Reply:
belongs_to thiết lập kết nối one-to-one (1-1) giữa model micropost với model user. ở đây 1 micropost chỉ thuộc một user

Reply:
chưa chuẩn em nhé
user - micropost là mqh 1 - n (one-to-many)
Tìm hiểu lại phần này

Reply:
dạ vâng, nhưng ở đây em đang đứng ở model micropost. 

tại model micropost:
belongs_to :user ----(1 micropost - 1 user)--------(như vậy thì 1 micropost chỉ có 1 user)

tại model user:
has_many :micropost (1 user - n micropost) ------( và 1 user sẽ có nhiều micropost)

chắc chị đang hiểu sai ý của em mô tả